news/2021-01-09-new-featured-artist-phantom-sage.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "New Featured Artist: Phantom Sage"
+date: 2021-01-09 09:00:00 +0000
+---
+
+We're proud to introduce **Phantom Sage** as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/129/header.jpg)
+
+Sage Douglas began producing music in his early teens under the stage name [**Phantom Sage**](https://osu.ppy.sh/beatmaps/artists/129) as a journey through the boundaries between EDM and ambient soundscapes. Rising to prominence with releases on the online label sensation *NoCopyrightSounds*, his works have made an impact on countless individuals throughout the last half-decade.
+
+Taking a step away from electronic music production in recent time, **Phantom Sage** has sprung to fame on TikTok as [*fntmsage*](https://www.tiktok.com/@fntmsage), where he covers songs throughout the genre spectrum for his 1.8 million followers.
+
+Celebrating his older works here on osu!, we've snagged a collection of **10** songs for your choosing, including classics like *Holystone* and *MIKO*.
+
+Not familiar with **Phantom Sage**'s songs? Worry not — [WillCookie](https://osu.ppy.sh/users/6404488) is here to show you how a top player manages on a map of Sage's music (albiet with some handicaps).
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/129/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Holystone
+
+Give a play to [this osu! set](https://osu.ppy.sh/beatmapsets/515297) by [sheela](https://osu.ppy.sh/users/1138027) or [this osu! set](https://osu.ppy.sh/beatmapsets/514942) by [Nozhomi](https://osu.ppy.sh/users/2716981), both part of the 2nd French Beatmapping Contest!
+
+Or if you're a keyboard smasher, [UnluckyCroco](https://osu.ppy.sh/users/9461160)'s [osu!mania mapset](https://osu.ppy.sh/beatmapsets/1001794) may be a better choice.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/129/Holystone/Phantom%20Sage%20-%20Holystone.mp3" type="audio/mpeg">
+</audio>
+
+### MIKO
+
+Or check out [this osu!mania mapset](https://osu.ppy.sh/beatmapsets/453386) for a more relaxing experience!
```

Comment:
suggestion :

``Or check out [this osu!mania mapset](https://osu.ppy.sh/beatmapsets/453386) by [Protastic101](https://osu.ppy.sh/users/6712747) for a more relaxing experience!``

Reply:
added